help/destinations/catalog/crm/salesforce.md

```diff
@@ -255,11 +279,13 @@ All [!DNL Adobe Experience Platform] destinations are compliant with data usage
 
 ### Unknown errors encountered while pushing events to the destination {#unknown-errors}
 
-When checking a dataflow run, if you obtain the following error message: `Unknown errors encountered while pushing events to the destination. Please contact the administrator and try again.`
+* When checking a dataflow run, if you obtain the following error message: `Unknown errors encountered while pushing events to the destination. Please contact the administrator and try again.`
+    ![Platform UI screenshot showing error.](../../assets/catalog/crm/salesforce/error.png)
 
-![Platform UI screenshot showing error.](../../assets/catalog/crm/salesforce/error.png)
+    * To fix this error, verify that the **[!UICONTROL Mapping ID]** you provided in [!DNL Salesforce CRM] for your Platform segment is valid and exists within [!DNL Salesforce CRM].
 
-To fix this error, verify that the **[!UICONTROL Mapping ID]** you provided in [!DNL Salesforce CRM] for your Platform segment is valid and exists within [!DNL Salesforce CRM].
+* When activating a segment, if you obtain an error message: `The client's IP address is unauthorized for this account. Allowlist the client's IP address...`
```

Comment:
See my suggested verbiage for this section within the other PR that you opened. Thanks!